lib/tasks/rummager.rake

```diff
@@ -45,6 +45,25 @@ namespace :rummager do
       end
       puts "Complete."
     end
+
+    desc "indexes all content belonging to a model"
```

Comment:
I'll look into it - we don't need batching for topical events since there are so few, but if there's an existing thing that does the job there's no point adding another one.